File: Assignment1/consExcSum.java
import java.util.*;

public class consExcSum
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter number: ");
		int n = sc.nextInt();
		int sum = 0;
		while (n>9)
		{
			int b = n%10;
			n /= 10;
			int a = n%10;
			sum += ((b*10)+a);
		}
		System.out.println(sum);

		sc.close();
	}
}

File: Assignment1/consExcSum.cpp
#include <iostream>
using namespace std;

int main() {
    int n, sum = 0;
    cout << "Enter number: ";
    cin >> n;
    while (n > 9) {
        int b = n % 10;
        n /= 10;
        int a = n % 10;
        sum += ((b * 10) + a);
    }
    cout << sum << endl;
    return 0;
}